src/middlewared/middlewared/utils/__init__.py

```diff
@@ -134,6 +137,60 @@ def setusercontext(user):
         os.chdir('/')
 
 
+def _run_command(user, commandline, q, rv, log_ident):
+    setusercontext(user)
+
+    os.environ['PATH'] = (
+        '/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:/usr/local/sbin:/root/bin'
+    )
+    proc = subprocess.Popen(
+        commandline, shell=True, stdout=subprocess.PIPE,
+        stderr=subprocess.STDOUT
+    )
+
+    # logger(1) does not honor original exit value so we use syslog module instead
+    syslog.openlog(ident=log_ident)
```

Comment:
We used to use syslog because there was no way to retrieve a specific job output.
I am thinking it should be OK to just drop syslog now in favor of job log? Thoughts?